front/app/components/AddEventToCalendarButton/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+// components
+import { Box, Text, Button, Error } from '@citizenlab/cl2-component-library';
+
+// intl
+import messages from './messages';
+import { useIntl } from 'utils/cl-intl';
+
+// util
+import { requestBlob } from 'utils/request';
+import { API_PATH } from 'containers/App/constants';
+import saveAs from 'file-saver';
+
+export const AddEventToCalendarButton = ({ eventId }: { eventId: string }) => {
+  const { formatMessage } = useIntl();
+  const [isError, setIsError] = useState(false);
```

Comment:
This works, but now once you have the error it will never be unset. Maybe you should set it back ot `false` at the beginning of `handleICSDownload`?

Reply:
Or if you think this behavior is fine feel free to merge!

Reply:
Yes you're completely right. Thanks for catching that :pray: I went too quickly when making the change.